scripts/services/migrations.ts

```diff
@@ -88,6 +88,20 @@ export const migration: T.ExpectedExports.migration = async (
           );
         },
       },
+      "26.0.8.2": {
+        up: compat.migrations.updateConfig(
+          (_) => ({
+            "webdav": "1024",
```

Comment:
this doesn't look right

Reply:
It does migrate properly, but throws an error (`No Match: webdav: Field Is Not Nullable`) - I'm not sure why, this is identical to the previous migration for default locales.  Does it need to be passed `config` instead of `_`?

Reply:
This overwrites the entire config, rather than setting just the value you want. I don't think that's what you want. Try something like `config => { config.webdav = 1024; return config }`